src/constants/ActionConstants.js

```diff
@@ -142,3 +142,8 @@ export const DISMISS_MESSAGE = "DISMISS_MESSAGE";
 export const LOAD_RECORDS_PHASES_PENDING = "LOAD_PHASES_PENDING";
 export const LOAD_RECORDS_PHASES_SUCCESS = "LOAD_PHASES_SUCCESS";
 export const LOAD_RECORDS_PHASES_ERROR = "LOAD_PHASES_ERROR";
+
+// ActionConstants.js
```

Comment:
```suggestion
```

Reply:
Why adding this string? Does it say something important?

Reply:
Maybe instead I would add the comment:

`// server configuration`

The reason is that I have a little issue with naming `LOAD_ALLOWED_REJECT_MESSAGE_`... as it is a little different from other actions as it is related to the configuration of the server. Not sure if name-ing should solve it but since this is "first of this kind" ActionConstant i suggest to add comment at least to create "a section". Later, we can decide if we should use some name-ing convention here.